content/blog/foundry_vtt.md

```diff
@@ -0,0 +1,176 @@
+
+#### Quick Tip for Testing (Optional):
+
+If you want to test Pinggy without launching Foundry first, you can quickly spin up a basic local web server using Python. This is a handy way to confirm everything is working correctly:
+
+Open your terminal or command prompt, and run:
+
+```bash
+python3 -m http.server 3000
+```
+
+This will create a simple web server accessible at `http://localhost:3000`.
+
+### Step 2: Create a Public URL Using Pinggy (Single SSH Command)
+
+Next, you’ll create a public tunnel to your local Foundry VTT instance using Pinggy’s single-line SSH command.
+
+#### Execute the Following Command:
+
+Open your terminal (macOS/Linux) or PowerShell (Windows) and run:
+
+```bash
+ssh -p 443 -R0:localhost:3000 a.pinggy.io
+```
+
+{{< ssh_command defaultcommand="ssh -p 443 -R0:localhost:3000 a.pinggy.io" >}}
+"{\"cli\":{\"windows\":{\"ps\":\"./pinggy.exe -p 443 -R0:localhost:3000 a.pinggy.io\",\"cmd\":\"./pinggy.exe -p 443 -R0:localhost:3000 a.pinggy.io\"},\"linux\":{\"ps\":\"./pinggy -p 443 -R0:localhost:3000 a.pinggy.io\",\"cmd\":\"./pinggy -p 443 -R0:localhost:3000 a.pinggy.io\"}},\"ssh\":{\"windows\":{\"ps\":\"ssh -p 443 -R0:localhost:3000 a.pinggy.io\",\"cmd\":\"ssh -p 443 -R0:localhost:3000 a.pinggy.io\"},\"linux\":{\"ps\":\"ssh -p 443 -R0:localhost:3000 a.pinggy.io\",\"cmd\":\"ssh -p 443 -R0:localhost:3000 a.pinggy.io\"}}}"
+{{</ ssh_command >}}
+
+{{< image "foundry_vtt/pinggy_command.webp" "Pinggy Command" >}}
+
+**Explanation of this command:**
+
+- `-p 443`: Uses port 443 for SSH, ensuring compatibility through most firewalls.
+- `-R0:localhost:3000`: Requests Pinggy to forward connections from a random public URL to your local port `3000`.
+- `a.pinggy.io`: Connects to Pinggy's tunneling server.
+
+#### Example Output:
+
+Once you execute the above command, Pinggy will automatically assign you a unique public URL. Your terminal output will look similar to:
+
+```bash
+You can access your local server via the following URL(s):
+http://abc123xyz.a.pinggy.link
+https://abc123xyz.a.pinggy.link
+```
+
+{{< image "foundry_vtt/pinggy_public_url.webp" "Pinggy Public Url" >}}
+
+#### How to Share with Players:
+
+Copy one of these provided URLs and share it directly with your Foundry VTT players. They can now instantly access your Foundry session from anywhere on the web—no additional setup required!
+
+## Advanced Features for Enhanced Management and Security (Optional, Recommended)
+
+Pinggy offers a few powerful features you can optionally leverage to enhance the security and manageability of your tunnels:
+
+### Manage All Your Active Tunnels Using Pinggy Dashboard
+
+Pinggy provides an intuitive, web-based dashboard to monitor and manage your tunnels, connections, and usage statistics.
+
+- Visit the [Pinggy Dashboard](https://dashboard.pinggy.io) and sign up or sign in.
+- Once logged in, you’ll receive a personalized **access token**.
+
+{{< image "foundry_vtt/pinggy_dashboard.webp" "Pinggy Dashboard" >}}
```

Comment:
the image contains your access token. please reset the access token in your account.